com.unity.render-pipelines.universal/Editor/UniversalRenderPipelineAnimationClipUpgrader.cs

```diff
@@ -0,0 +1,31 @@
+using System.Collections.Generic;
+
+namespace UnityEditor.Rendering.Universal
+{
+    class UniversalRenderPipelineAnimationClipUpgrader
+    {
+        [MenuItem("Edit/Rendering/AnimationClips/Convert All AnimationClips to URP Properties")]
+        static void UpgradeAllAnimationClipsMenuItem()
+        {
+            var upgraders = new List<MaterialUpgrader>();
+            UniversalRenderPipelineMaterialUpgrader.GetUpgraders(ref upgraders);
+            AnimationClipUpgrader.DoUpgradeAllClipsMenuItem(upgraders);
+        }
```

Comment:
Instead of adding a menu we should integrate this to the material upgrade as discussed.

Reply:
Done in https://github.com/Unity-Technologies/Graphics/pull/4135/commits/5aa6524419a08dd468f9507f56f10ce9c2aea806. To keep changes minimal for now, it is still a second dialog prompt that appears, but as part of the existing menu command.